.github/workflows/test.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       - uses: actions/checkout@v3
```

Comment:
Is the comment above on line 35 still relevant?

Reply:
Good point, I updated the comment. Arbitrary requires 1.63.